ubuntudesign/cn.ubuntu.com
# add ubuntu core 16 launch news

## Done

* added a new news article

## QA

1. go to /news and see the new IOT article
2. go to /news/2016/11/18/ubuntu-core-16-launches/ and see that the doc looks ok
3. compare to the [copy doc](https://docs.google.com/document/d/1afpFOYSzmTlgibXrRxZNaK7gCMjQXhDCkEZV6aBVnBE/edit?ts=582462cd#)

## Issue / Card

[trello card](https://trello.com/c/KBLxyXfx)
